components/brave_policy/common/BUILD.gn

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("common") {
+  sources = [ "brave_policy_observer.h" ]
+
+  deps = [ "//base" ]
```

Comment:
also fyi any header dependencies should be public_deps

Reply:
changed, thanks!